service/controller/resource/nodepool/template/main.json

```diff
@@ -261,15 +267,17 @@
       "tags": {
         "provider": "[toUpper(parameters('GiantSwarmTags').provider)]",
         "cluster-autoscaler-enabled": "true",
```

Comment:
Would it make sense to reflect this flag based on whether specific `MachinePool` has `min` / `max` annotations set or not? Wouldn't `cluster-autoscaler` otherwise always auto-scale the VMSS?

Reply:
It makes sense to enable it in any case. If the node pool has min == max set, the auto scaler will simply not scale that node pool. If the customer updates the min or max, then it will just work immediately which makes sense to me.